src/include/daos/policy.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * (C) Copyright 2021-2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * This file is part of daos
+ *
+ * include/daos/policy.h
+ */
+
+#ifndef __DAOS_POLICY_H__
+#define __DAOS_POLICY_H__
+#include <daos/common.h>
+
+/**
+ * tier placement policy
+ */
+typedef enum tier_policy {
+        TIER_POLICY_NOT_EXIST = -1,
+        TIER_POLICY_DEFAULT,
```

Comment:
What's the DEFAULT policy? Isn't it one of the following policies you listed?

Just FYI, we had some planed policies in NVMe support design phase (not implemented):
- IO characteristic: Selecting media by io size;
- SCM caching: Absorbing all front IOs by SCM, migrating to NVMe by background aggregation;
- User driven: User provide hints by DAOS API to influence media selection;
- Auto tuning: analyze IO history data to predict IO workload and do proper media selection;